fe/fe-core/src/main/java/com/starrocks/sql/parser/AstBuilder.java

```diff
@@ -539,6 +539,48 @@ public ParseNode visitEmptyStatement(StarRocksParser.EmptyStatementContext conte
         return new EmptyStmt();
     }
 
+    // ---------------------------------------- Warehouse Statement ---------------------------------------------------
+
+    @Override
+    public ParseNode visitCreateWarehouseStatement(StarRocksParser.CreateWarehouseStatementContext context) {
+        throw new SyntaxNotSupportException(context.start, context.stop);
```

Comment:
‌‌‌‌‌‌Just a question, taking the CreateWarehouse Statement as an example, which part of the string corresponds to the values of context.start and context.end?